Helpers.py

```diff
@@ -854,38 +855,28 @@ def assign_color(sheet, row, column, camper):
 
 
 
-
-"""Output cycle excel document"""
-def output_cycle_excel(campers_list, title):
-    book = Workbook()           # Create the workbook
-    sheet = book.active         # Access the active sheet
-
-    # Write in explicit header row
-    sheet.cell(row=1, column=1).value = "Name"      # Note: rows and columns indexed starting from 1 in openpyxl
-    sheet.cell(row=1, column=2).value = "Edah"
-    sheet.cell(row=1, column=3).value = "Tzrif"
-    sheet.cell(row=1, column=4).value = "Peulah"
-
-    # Write in camper information relevant to the cycle
-    for i in range(len(campers_list)):
-        sheet.cell(row=i + 2, column=1).value = campers_list[i].name
-        sheet.cell(row=i + 2, column=2).value = campers_list[i].edah
-        sheet.cell(row=i + 2, column=3).value = campers_list[i].bunk
-        sheet.cell(row=i + 2, column=4).value = campers_list[i].next_activity
-
-
-
 # ===================================================================================================
 
 
 
-"""Output master excel document"""
-def output_master_excel(campers_list):
-    book = Workbook()           # Create the workbook
-    sheet = book.active         # Access the active sheet
+def output_master_excel(campers_list, activities_list):
+    """Output master excel document"""
+    # Create the workbook
+    book = Workbook()
+    # Access the active sheet
+    sheet = book.active
+    book.sheetnames[0] = "Bechirot"
 
-    # Write in explicit header row
-    sheet.cell(row=1, column=1).value = "Name"      # Note: rows and columns indexed starting from 1 in openpyxl
+    # Sets column width for columns 1 and 4
+    column = 1
+    while column < 5:
+        if column == 1 or column == 4:
+            i = get_column_letter(column)
+            sheet.column_dimensions[i].width = 23
+        column += 1
+
+    # Write in explicit header row -- Note: rows and columns indexed starting from 1 in openpyxl
+    sheet.cell(row=1, column=1).value = "Name"
     sheet.cell(row=1, column=2).value = "Edah"
     sheet.cell(row=1, column=3).value = "Tzrif"
     sheet.cell(row=1, column=4).value = "Peulah"
```

Comment:
In the part below here, you should just do something like
```python
max_past_activities = max(len(camper.past_activities) for camper in campers_list)
```
and also probably use `enumerate` in the part below that.